src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -308,6 +308,69 @@ newrelic-infrastructure:
   Setting `interval` to values larger than `40s` is not allowed.
 </Callout>
 
+### Filtering Namespaces [#filter-namespace]
+
+The Kubernetes Integration v3 and above allows filtering which namespaces will be scraped by allowing you to include or exclude namespaces based on labels. By default all namespaces will be scraped.
+
+In order to include only namespaces matching a label, change the `namespaceSelector` by adding the following to your `values-newrelic.yaml`, under the `newrelic-infratructure` section:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
```

Comment:
Could be just mention that the `namespaceSelector` syntax and behaviour is the K8s one?

Reply:
Yes, I'll add it

Reply:
done